lib/react-carousel.js

```diff
@@ -3,9 +3,9 @@
"+o+" `"+a+"` supplied to `"+r+"`.")}if(!Array.isArray(t))return"production"!==e.env.NODE_ENV&&u("Invalid argument supplied to oneOfType, expected an instance of array."),r;for(var o=0;o<t.length;o++){var a=t[o];if("function"!=typeof a)return u("Invalid argument supplied to oneOfType. Expected an array of check functions, but received "+E(a)+" at index "+o+"."),r}return f(n)}function b(t){function e(e,n,r,o,a){var u=e[n],s=x(u);if("object"!==s)return new l("Invalid "+o+" `"+a+"` of type `"+s+"` supplied to `"+r+"`, expected `object`.");for(var c in t){var f=t[c];if(f){var p=f(u,c,r,o,a+"."+c,i);if(p)return p}}return null}return f(e)}function m(t){function e(e,n,r,a,u){var s=e[n],c=x(s);if("object"!==c)return new l("Invalid "+a+" `"+u+"` of type `"+c+"` supplied to `"+r+"`, expected `object`.");var f=o({},e[n],t);for(var p in f){var d=t[p];if(!d)return new l("Invalid "+a+" `"+u+"` key `"+p+"` supplied to `"+r+"`.\nBad object: "+JSON.stringify(e[n],null,"  ")+"\nValid keys: "+JSON.stringify(Object.keys(t),null,"  "));var h=d(s,p,r,a,u+"."+p,i);if(h)return h}return null}return f(e)}function w(e){switch(typeof e){case"number":case"string":case"undefined":return!0;case"boolean":return!e;case"object":if(Array.isArray(e))return e.every(w);if(null===e||t(e))return!0;var n=s(e);if(!n)return!1;var r,o=n.call(e);if(n!==e.entries){for(;!(r=o.next()).done;)if(!w(r.value))return!1}else for(;!(r=o.next()).done;){var i=r.value;if(i&&!w(i[1]))return!1}return!0;default:return!1}}function _(t,e){return"symbol"===t||("Symbol"===e["@@toStringTag"]||"function"==typeof Symbol&&e instanceof Symbol)}function x(t){var e=typeof t;return Array.isArray(t)?"array":t instanceof RegExp?"object":_(e,t)?"symbol":e}function O(t){if(void 0===t||null===t)return""+t;var e=x(t);if("object"===e){if(t instanceof Date)return"date";if(t instanceof RegExp)return"regexp"}return e}function E(t){var e=O(t);switch(e){case"array":case"object":return"an "+e;case"boolean":case"date":case"regexp":return"a "+e;default:return e}}function C(t){return t.constructor&&t.constructor.name?t.constructor.name:T}var P="function"==typeof Symbol&&Symbol.iterator,j="@@iterator",T="<<anonymous>>",S={array:p("array"),bool:p("boolean"),func:p("function"),number:p("number"),object:p("object"),string:p("string"),symbol:p("symbol"),any:function(){return f(r)}(),arrayOf:d,element:function(){function e(e,n,r,o,i){var a=e[n];if(!t(a)){return new l("Invalid "+o+" `"+i+"` of type `"+x(a)+"` supplied to `"+r+"`, expected a single ReactElement.")}return null}return f(e)}(),instanceOf:h,node:function(){function t(t,e,n,r,o){return w(t[e])?null:new l("Invalid "+r+" `"+o+"` supplied to `"+n+"`, expected a ReactNode.")}return f(t)}(),objectOf:y,oneOf:v,oneOfType:g,shape:b,exact:m};return l.prototype=Error.prototype,S.checkPropTypes=a,S.PropTypes=S,S}}).call(e,n(10))},function(t,e,n){"use strict";function r(t){if(null===t||void 0===t)throw new TypeError("Object.assign cannot be called with null or undefined");return Object(t)}/*
```

Comment:
IMO this file should be build in prepublish (and publishing should be in CI);

however as it was already here, I'm not blocking this PR